### PR TITLE
update to scalatest 3.0.0, sbt 0.13.12, and scala 2.11.8

### PR DESCRIPTION
**Pull request by dvic**

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -24,7 +24,7 @@ import sbt.inc.Analysis
 
 object BuildSettings {
   val buildVersion = "3.2.2"
```
Comment (fommil) [inline project/Build.scala]: this might need bumped

**Comment by damirv** [comment]
@fommil I added a commit in which I bumped the version to 3.3.0

**Comment by ngbinh** [comment]
can any collaborator review and merge this? Thanks

**Comment by smessmer** [comment]
+1

**Comment by dvic** [comment]
Hmm, does anyone know what is the status of this project? It feels a bit abandoned (with the last commit over a year ago).

**Comment by fommil** [comment]
@paulbutcher will get to it, but the weather has been fantastic in the UK these last few days so just be patient and use the workaround above :wink:

**Comment by dvic** [comment]
Ah okay, glad to hear that :) Enjoy the weather 👍

**Comment by Maatary** [comment]
Hi Any news on that front ?

**Comment by nartamonov** [comment]
+1 realy wanted :)

**Comment by grubino** [comment]
+1

**Comment by Scalar4eg** [comment]
+1, please

**Comment by xavier-fernandez** [comment]
+1

**Comment by ssemichev** [comment]
+1

**Comment by dvic** [comment]
@fommil @paulbutcher I know you guys are very busy, but do you have any idea when you could merge this 😉 ?

**Comment by fommil** [comment]
@dvic why are you asking me?

**Comment by dvic** [comment]
@fommil Apologies, I thought you were also an active contributor (or in touch with Paul).

**Comment by fommil** [comment]
@dvic :stuck_out_tongue:

**Comment by nastich** [comment]
@dvic have you checked if your version works with new Async suites?

**Comment by dvic** [comment]
@nastich Yes, I did not have any problems with `AsyncSuite` in my tests, although I might want to add some tests for those in this PR.

**Comment by YuvalItzchakov** [comment]
@dvic Hi, did this end up being merged?

**Comment by dvic** [comment]
@YuvalItzchakov No haven't heard anything yet.

**Comment by SethTisue** [comment]
this would be nice to have for the Scala 2.12 community build as well, which we've recently moved to ScalaTest 3.0

**Comment by Tolsi** [comment]
+1

**Comment by cheeseng** [comment]
FYI: I merged this into master and published 3.3.0 for Scala 2.11.  Hope it helps.

**Comment by fommil** [comment]
Thanks! Can you do one for 2.10 as well please?

**Comment by cheeseng** [comment]
@fommil I am not sure how different is master-2.10 than master, do you mind submitting a PR against master-2.10?

**Comment by fommil** [comment]
Sorry I'm on holiday, no computer. Maybe somebody in this thread could do it?

**Comment by SethTisue** [comment]
thank you!

**Comment by kevincoghlan** [comment]
PR created against the Scala 2.10 branch, containing @dvic 's ScalaTest 3.0.0 compatibility changes: https://github.com/paulbutcher/ScalaMock/pull/154

Could someone look it over? If it's OK, perhaps @cheeseng could merge & publish?

Thanks folks.

**Comment by cheeseng** [comment]
FYI: Merged and ScalaMock 3.3.0 built for Scala 2.10 published in sonatype.

**Comment by kevincoghlan** [comment]
Thank you @cheeseng !

**Pull request merged**